pg/benchmarks/pg_suite_runner.rb

```diff
@@ -40,8 +43,69 @@ def run_benchmark(bm_script)
   end
 
   def execute(command)
-    result = JSON.parse(`#{command}`)
-    puts "[#{result['label']}] #{result['iterations_per_second']} ips, #{result['total_allocated_objects_per_iteration']} objects"
-    result
+    output = JSON.parse(`#{command}`)
+    puts "[#{output['label']}] #{output['iterations_per_second']} ips, #{output['total_allocated_objects_per_iteration']} objects"
+    output
+  end
+
+  def send_results(output, bm_script)
+    form_data = default_form_data(output, bm_script)
+
+    send_ips(output, form_data)
+    send_objects(output, form_data)
+  end
+
+  def default_form_data(output, bm_script)
+    {
+      'benchmark_type[category]' => output['label'],
+      'benchmark_type[script_url]' => "#{RAW_URL}#{Pathname.new(bm_script).basename}",
+      'benchmark_type[digest]' => generate_digest(bm_script),
+      'benchmark_run[environment]' => "#{`ruby -v`}",
```

Comment:
And if you're sure you're running in the same Ruby, you can use the RUBY_VERSION constant, though that's not 100% equivalent to the above -- depends if you might be running in one Ruby while another is in the path :-)